lindera/src/formatter.rs

```diff
@@ -1,41 +1,45 @@
+use lindera_core::error::LinderaErrorKind;
+use lindera_core::LinderaResult;
+
 use crate::tokenizer::Token;
 
 #[derive(Debug, Clone, Copy)]
 pub enum Format {
     MeCab,
     Wakati,
-    JSON,
+    Json,
```

Comment:
We should use JSON I think. Or we should change MeCab to Mecab.

Reply:
Changed `JSON` to `Json` according to the naming convention.
https://doc.rust-lang.org/1.0.0/style/style/naming/README.html